cfme/fixtures/appliance.py

```diff
@@ -116,8 +116,8 @@ def temp_appliances_unconfig_modscope():
         yield appliances
 
 
-@pytest.fixture(scope="module")
-def temp_appliances_unconfig_modscope_rhevm():
+@pytest.yield_fixture(scope="function")
```

Comment:
Required: Please use fixture instead of yield_fixture as it was deprecated
@pytest.fixture, by default this is function scope